.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,22 @@
+name: build-docker
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
+
+jobs:
+  build:
```

Comment:
Please, name this job to `build-docker`. Unfortunately in settings only job names are displayed (without workflow names), so I see just `build`